#! /bin/sh /usr/share/dpatch/dpatch-run
## 04-makefile-strip.dpatch by Luca Bedogni <me@lucabedogni.it>
##
## DP: Delete automatic stripping of binary

@DPATCH@

--- wmsysmon-0.7.7/Makefile	2007-08-16 16:15:32.000000000 +0200
+++ wmsysmon-0.7.7/Makefile	2007-08-16 16:14:27.000000000 +0200
@@ -1,7 +1,7 @@
 CC = gcc
 
 LIBDIR += -L/usr/X11R6/lib
-LDFLAGS += -lXpm -lXext -lX11 -lm -s
+LDFLAGS += -lXpm -lXext -lX11 -lm
 #CFLAGS = -g #-DMONDEBUG
 CFLAGS += -Wall -O3 -fomit-frame-pointer -I/usr/X11R6/include #-DHI_INTS #undefine HI_INTS if your x86 SMP or alpha
 
